src/react-navigation.web.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
 
   // Navigators
   get createNavigator() {
-    return require('./navigators/createNavigator').default;
+    return require('./navigators/createNavigatorDEPRECATED').default;
```

Comment:
is this intentional?

Reply:
(this file doesn't seem to exist)

Reply:
whoops, will fix. At some point I had just renamed it, but we do want to change the external API here.

Reply:
ping @ericvicenti  ;)